doc/assets/cover.tmpl

```diff
@@ -1,16 +1,28 @@
+.. class:: logoimage
+
+.. image:: assets/rst2pdf-logo.jpg
+   :width: 50%
+
+
 .. raw:: pdf
 
-   Spacer 0 73mm
+   Spacer 0 10mm
+
+
+.. class:: subtitle
+
+How to use
 
 .. class:: title
 
-{{title}}
+rst2pdf
```

Comment:
Was this intentional?

Reply:
Yes, this is intentional. I've split the title on the cover page over two lines with different font sizes.